app-text/kbibtex/kbibtex-9999.ebuild

```diff
@@ -47,6 +46,15 @@ DEPEND="
 "
 RDEPEND="${DEPEND}
 	dev-tex/bibtex2html
+	!app-text/kbibtex:4
 "
 
 S=${WORKDIR}/${P/_/-}
+
+pkg_pretend() {
+	if [[ ${MERGE_TYPE} != binary  && $(tc-getCC) == *gcc* ]]; then
```

Comment:
If I understand @mgorny correctly, the version check will work as expected with clang thus is not necessary to check for gcc.